wurf_android_copy_lib_to_app.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""Tool for copying jni libraries to the appropriate Android app folder."""
+
+import os
+import shutil
+from waflib.TaskGen import feature
+from waflib.TaskGen import after_method
+
+from waflib import Task, Errors, Logs
+
+
+@feature('android_copy_lib_to_app')
+@after_method('apply_link')
+def android_copy_lib_to_app(self):
+    """
+    Copy library to Android application.
+
+    When building an Android application from the IDE native libraries placed
+    in the libs folder will be packaged in the apk. So we copy the shared libs
+    we build there.
+    """
+    if not self.bld.is_mkspec_platform('android'):
+        return
+
+    input_librarys = self.link_task.outputs
+    output_librarys = []
+    for input_library in input_librarys:
+        project_path = self.path.parent.abspath()
+        output_library = os.path.basename(input_library.abspath())
+        output_library = self.bld.root.make_node(
+            os.path.abspath(os.path.join(
+                project_path,
+                'app',
```

Comment:
I found the relevant description for `export_includes`: https://github.com/waf-project/waf/blob/9625343f91810ffeb8b844f14f997daf44a91ff5/waflib/Tools/ccroot.py#L66
Clearly, the paths are relative and we have a mistake here: the wscript_build sits in a subfolder, so the path should be `../jni` or just `.` (but nobody uses the exported includes, so it does not matter here).

We should make the copy_path relative and we can use `self.path.make_node(os.path.join(self.copy_path, output_library))` to get the output node.

I should note that the resolution of project_path is also bogus here:
`project_path = self.path.parent.abspath()`
This only works if we are in a direct subfolder of the project root.
The correct solution would be: `self.bld.srcnode`, but we don't need this with the relative paths!